python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -259,16 +259,22 @@ def refresh_progress_bar(self, resource_manager: ResourceManager) -> None:
     def summary_str(self, resource_manager: ResourceManager) -> str:
         queued = self.num_queued() + self.op.internal_queue_size()
         active = self.op.num_active_tasks()
-        desc = f"- {self.op.name}: {active} active, {queued} queued"
+        desc = f"- {self.op.name}: Tasks: {active}"
         if (
             self.op._in_task_submission_backpressure
             or self.op._in_task_output_backpressure
         ):
-            desc += " 🚧"
-        desc += f", [{resource_manager.get_op_usage_str(self.op)}]"
+            desc += " [backpressured]"
+
+        # Add operator suffix, which includes actor information,
+        # directly after task details.
         suffix = self.op.progress_str()
         if suffix:
             desc += f", {suffix}"
+
+        desc += f", Queued Blocks: {queued}; "
```

Comment:
Nit: for consistency with "backpressured" and "object store"
```suggestion
        desc += f", Queued blocks: {queued}; "
```